.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: cpp
+dist: trusty
+matrix:
+  include:
+  - os: linux
+    env: CONDA=3.6 RDKIT=2019.09.2
+  - os: linux
+    env: CONDA=3.7 RDKIT=2019.09.2
+  - os: linux
+    env: CONDA=3.8 RDKIT=2019.09.2
+  - os: osx
+    env: CONDA=3.6 RDKIT=2019.09.2
+  - os: osx
+    env: CONDA=3.7 RDKIT=2019.09.2
+  - os: osx
+    env: CONDA=3.8 RDKIT=2019.09.2
+
+before_install:
+- |
+  if [ "$TRAVIS_OS_NAME" = "linux" ]; then export CXX=g++-4.8 CC=gcc-4.8; fi
+  if [ -n "$PYTHON" ]; then
+    if [ "$TRAVIS_OS_NAME" = "osx" ] && [ "${PYTHON:0:1}" = "3" ]; then
+      brew update; brew install python3;
+    fi
+    pip install --user --upgrade pip virtualenv
+    virtualenv -p python$PYTHON venv
+    source venv/bin/activate
+  elif [ -n "$CONDA" ]; then
+    if [ "$TRAVIS_OS_NAME" = "linux" ]; then OS=Linux-x86_64; else OS=MacOSX-x86_64; fi
+    wget -O miniconda.sh https://repo.continuum.io/miniconda/Miniconda${CONDA:0:1}-latest-$OS.sh
```

Comment:
It's worth thinking about picking a single "supported" miniconda version and using that here. Perhaps instead of testing three different python versions (see above) we could test two miniconda versions: the "supported" one and "latest".

Reply:
nice idea! I'll then keep py3.8 with two conda versions :)